io.emeric.watchflower.yml

```diff
@@ -0,0 +1,98 @@
+app-id: io.emeric.watchflower
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: watchflower
+
+rename-desktop-file: watchflower.desktop
+rename-appdata-file: watchflower.appdata.xml
+#rename-icon: watchflower
+
+finish-args:
+  # Our UI is GPU accelerated
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # We need Bluetooth support
+  - --allow=bluetooth
+  - --system-talk-name=org.bluez
+  - --share=network
+
+cleanup:
+- /bin/__pypache__
+- /bin/rst*
+- /include
+- /lib/cmake
+- /lib/cups
+- /lib/pkgconfig
+- /lib/python*
+- /share/doc
+- /share/man
+- /share/zsh
+- /src
+- '*.a'
+- '*.la'
+
+modules:
+
+- name: python3-docutils # required to build bluez
+  buildsystem: simple
+  sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/4c/17/559b4d020f4b46e0287a2eddf2d8ebf76318fd3bd495f1625414b052fdc9/docutils-0.17.1.tar.gz
+      sha256: 686577d2e4c32380bb50cbb22f575ed742d58168cee37e99117a854bcd88f125
+  build-commands:
+    - pip3 install --no-index --find-links="file://${PWD}" --prefix="${FLATPAK_DEST}" docutils
+
+- name: bluez # required by qtconnectivity
+  config-opts:
+    - --disable-datafiles
+    - --disable-systemd
+    - --enable-experimental
+    - --enable-library
+    - --disable-client
+    - --disable-mesh
+    - --disable-tools
+    - --disable-monitor
+    - --disable-udev
+    - --prefix=/app
+    - --sysconfdir=/app/etc
+  sources:
+    - type: archive
+      url: https://www.kernel.org/pub/linux/bluetooth/bluez-5.62.tar.xz
+      sha256: 38090a5b750e17fc08d3e52178ed8d3254c5f4bd2c48830d5c1955b88e3bc0c2
+      x-checker-data:
+        type: anitya
+        project-id: 10029
+        url-template: https://www.kernel.org/pub/linux/bluetooth/bluez-$version.tar.xz
+
+- name: qtconnectivity
+  buildsystem: simple
+  cleanup-platform:
+    - /bin
+    - /mkspecs
+  sources:
+    - type: git
+      url: https://github.com/qt/qtconnectivity
+      tag: v5.15.2
+      commit: ca6cc606d9fc0947ea6c27738a1ca8f12f3258ea
+      x-checker-data:
+        type: anitya
+        project-id: 153467
+        tag-template: v$version
+  build-commands:
+    - qmake
+    - make -j $FLATPAK_BUILDER_N_JOBS
+    - cp -r -n {bin,include,lib} /app # we need to deploy libs at the right place
+    - mkdir -p /app/src/bluetooth
+    - cp -r src/bluetooth /app/src/ # header files too
```

Comment:
you could use `buildsystem: qmake` and add the rest into the `post-install` phase.